--- ocp-0.1.16/desktop/opencubicplayer.desktop.in~	2009-01-06 09:05:03.000000000 +0100
+++ ocp-0.1.16/desktop/opencubicplayer.desktop.in	2009-01-06 09:05:03.000000000 +0100
@@ -1,9 +1,10 @@
 [Desktop Entry]
 Name=Open Cubic Player
-Comment=Open Cubic Player @PACKAGE_VERSION@ for UNIX 
+Comment=Open Cubic Player for UNIX 
 Encoding=UTF-8
-Exec=@BINDIR@/ocp
+Exec=ocp
 Icon=opencubicplayer
 Terminal=false
 Type=Application
-Categories=Application;AudioVideo;
+Categories=AudioVideo;Player;
+# vi: encoding=utf-8
