--- ocp-0.1.17/Makefile.in~	2009-06-27 12:15:43.000000000 +0200
+++ ocp-0.1.17/Makefile.in	2009-07-15 11:21:43.000000000 +0200
@@ -39,8 +39,8 @@
 	$(CP) AUTHORS BUGS COPYING CREDITS KEYBOARD_REMAPS SUID TODO "$(DESTDIR)$(PREFIX)/share/doc/ocp$(DIR_SUFFIX)"
 	mkdir -p "$(DESTDIR)$(PREFIX)/share/ocp$(DIR_SUFFIX)/data"
 	$(CP) ocp.hlp "$(DESTDIR)$(PREFIX)/share/ocp$(DIR_SUFFIX)/data"
-	mkdir -p "$(DESTDIR)$(PREFIX)/share/ocp$(DIR_SUFFIX)/etc"
-	$(CP) ocp.ini "$(DESTDIR)$(PREFIX)/share/ocp$(DIR_SUFFIX)/etc"
+	mkdir -p "$(DESTDIR)/etc"
+	$(CP) ocp.ini "$(DESTDIR)/etc"
 	mkdir -p "$(DESTDIR)$(LIBDIR)"
 	$(CP) libocp$(LIB_SUFFIX) "$(DESTDIR)$(LIBDIR)"
 	mkdir -p "$(DESTDIR)$(LIBDIR)/autoload"
